### ISSUE TITLE
Images with alternative text should be included in card and row reading order

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2386]
## Description

https://teams.microsoft.com/l/message/19:be8725e15a8c4cc5ae1bf5d67fad4b7f@thread.skype/1759317356369?tenantId=9744600e-3e04-492e-baa1-25ec245c6f10&groupId=bef9be07-03e4-4afb-8a79-f720a63e4c28&parentMessageId=1759317356369&teamName=Novum&channelName=WebCore-public&createdTime=1759317356369

* This apply to cards and rows
   * Card specs: https://www.figma.com/design/tKdPOfcUALzVIh5oizFbm7/%F0%9F%94%B8-Cards-Specs?m=auto&node-id=11138-19855&t=x1b5muX2vHaYSawE-1
   * List specs: https://www.figma.com/design/Be8QB9onmHunKCCAkIBAVr/%F0%9F%94%B8-Lists-Specs?m=auto&node-id=4615-10677&t=Zk1g6lnkz174KewL-1
* The order where the image should be concatenated in the label should be defined

## Tasks

- [x] Update specs
- [ ] Design documentation
- [ ] Android ticket
- [ ] iOS ticket
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2328)
